krabs/krabs/property.hpp

```diff
@@ -64,9 +64,17 @@ namespace krabs {
          */
         _TDH_IN_TYPE type() const;
 
+        /**
+         * <summary>
+         * Retrieves the Tdh type of the property.
```

Comment:
consider `Retrieves the Tdh out type of the property.` to disambiguate `out_type` from `type` above